target/linux/ipq806x/image/Makefile

```diff
@@ -88,6 +88,21 @@ define Device/avm_fritzbox-4040
 endef
 TARGET_DEVICES += avm_fritzbox-4040
 
+define Device/comfast_cf-e370ac
+	$(call Device/FitImage)
+	DEVICE_DTS := qcom-ipq4029-cf-e370ac
+	KERNEL_LOADADDR := 0x80208000
+	KERNEL_INSTALL := 1
+	KERNEL_SIZE := 4096k
+	IMAGE_SIZE := 26624k
+	BOARD_NAME := cf-e370ac
+	DEVICE_TITLE := COMFAST CF-E370AC
+	IMAGES := sysupgrade.bin
+	IMAGE/sysupgrade.bin := append-kernel | pad-to $$$${KERNEL_SIZE} | append-rootfs | pad-rootfs | append-metadata
+	DEVICE_PACKAGES := ath10k-firmware-qca4019 ipq-wifi-comfast_cf-e370ac
```

Comment:
is it typo? 4019 vs 4029

Reply:
Also, is it ipq40xx device or ipq806x device ?

Reply:
IPQ4029,

Reply:
so, why you change  target/linux/**ipq806x**/image/Makefile ?
and why firmware called ath10k-firmware-**qca4019** ?

Reply:
When I added this patch, ipq40xx is not created. And I will not update this path anymore, because it can used for developers with old OPENWRT/LEDE. Also I do not want to cost too much time on it.

Reply:
Anyway, Did you test it on real hardware ? I want to continue your work, rebase to master and so on.

Reply:
Yes, tested on real hardware,